docs/admin/upgrade.rst

```diff
@@ -1,45 +1,44 @@
 Upgrading Weblate
 =================
 
-Docker image upgrades
+Docker-image upgrades
```

Comment:
Why? Nobody seems ot use it this way.

Reply:
To distinguish between "Upgrades of the docker image", and "Image upgrades labeled "Docker"".
https://www.grammarly.com/blog/hyphen/
Used elsewhere in Weblate after the debate where I was made aware.